core/src/migrations/000099-addIncrementalToSchedules.ts

```diff
@@ -0,0 +1,36 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.addColumn("schedules", "incremental", {
+      type: DataTypes.BOOLEAN,
+      allowNull: true,
+      defaultValue: false,
+    });
+
+    const [incrementalSources]: [Record<string, any>[], unknown] =
+      await queryInterface.sequelize.query(
+        `SELECT * FROM "sources" WHERE type LIKE '%-import-table'`
+      );
+
+    if (incrementalSources.length > 0) {
+      await queryInterface.sequelize.query(
+        `UPDATE schedules SET incremental=true WHERE id IN (${incrementalSources
+          .map((s) => s.id)
+          .join(", ")})`
+      );
+    }
```

Comment:
The select statement above is looking for sources, whereas this update query is targeting schedules based on those IDs we just looked up. Did you mean to use WHERE sourceId here?